python/ray/serve/examples/doc/tutorial_tensorflow.py

```diff
@@ -1,19 +1,19 @@
+# This file is a copy of python/ray/serve/examples/doc/tutorial_tensorflow_full.py. If
+# you make any changes, please make the changes to both files.
```

Comment:
why are we making copies here?

Reply:
We wanted to include a copy of the full script at the end of the doc page so it's clearer to the reader, but the original file `tutorial_tensorflow.py` has all of the `__doc__` comment dividers so Shreyas and I decided to create a copy of the file. Do you have suggestions @simon-mo, we were also trying to think of other alternatives.

Reply:
It's harder to keep them in sync; imo it's not worth the maintenance burden

Reply:
Got it, will omit full script for now.